### PR TITLE
Dcip2d recipes

### PR DESCRIPTION
**Pull request by dccowan**
The following changes were made:

- Add recipe for forward modeling dcip2d data
- Add recipe for inverting dcip2d data
- Included in recipes are downloadable zip files containing an example
- Minor restructure of recipes section
- Add edit options (describe all parameters) for dcip2dfwd and dcip2dinv

### CODE REVIEW

File: content/objectFunctionality/forward/editOptions/dcip.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    	- *Value:* Set the surface to the specified elevation value
+    	- *Object:* A 2D topography data object
+
+    - **Conductivity Model:** Set the conductivity model (required for "DC2Dforward" and "IP2Dforward" objects)
+
+    - **Chargeability Model:** Set the chargeability model (not required for "DC2Dforward" objects)
+
+    - **Wave:** To solve the 2D problem, the problem must be solved in the wave domain. *N* specifies the number of log-distribution waves numbers used between *Min* and *Max*. The default is set to: *N* = 13, *Min* = 2.5e-4 and *Max* = 1.
+
+    - **Other options (IP only):** For IP forward modeling, the user may specify if the IP data are computed using a non-linear method or by a linear approximation (in which case the sensitivity is stored)
```
Comment (thast) [inline content/objectFunctionality/forward/editOptions/dcip.rst]: This option has implications that need to be at least mentioned here (there are detailed more in the manual):
IPL: you can invert chargability in whatever unit and you will get back a result with the same unit
IP non linear: IP signal is in percent %.

File: content/recipes/fwd/fwddcip2d.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    1. Load files (optional)
+    2. Predict DC data and load results
+    3. Predict IP data and load results
+
+**Required GIFtools Objects:**
+
+    - 2D tensor mesh
+    - 2D topography
+    - Locations
```
[the remark this answers is not in the document]
Comment (dccowan) [reply content/recipes/fwd/fwddcip2d.rst]: Good catch. It's more specific this way.

File: content/recipes/fwd/fwddcip2d.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+
+Predict DC Data and Load Results
+--------------------------------
+
+To predict DC data and load the results into GIFtools, carry out the following steps:
+
+    - :ref:`Create 2D DC forward modeling object <createDCIPFwd>`: The forward modeling object is created to provide a single item that 1) contains all parameters relevant to the forward model and 2) keeps track of which mesh, data locations etc... are being used in the forward model.
+
+    - :ref:`Set the forward modeling parameters through edit options <fwdEditOptions_dcip2d>`: Before we can predict data we must define the mesh, conductivity model, electrode locations, topography and other parameters. This is done through edit options.
```
[the remark this answers is not in the document]
Comment (dccowan) [reply content/recipes/fwd/fwddcip2d.rst]: These are already linked

File: content/recipes/fwd/fwddcip2d.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+
+.. figure:: ../images/dcip2d_true_model.png
+    :align: center
+    :width: 700
+
+    Conductivity (left) and chargeability (right) models download from .zip file
+
+
+Predict DC Data and Load Results
```
[the remark this answers is not in the document]
Comment (dccowan) [reply content/recipes/fwd/fwddcip2d.rst]: This should be obvious by clicking the links.

**Pull request merged**